website/source/docs/agent/configuration/server.html.md

```diff
@@ -139,20 +139,31 @@ server {
   useful for cases where we know the address will become available eventually.
   Use `retry_join` with an array as a replacement for `start_join`, **do not use
   both options**. See the [server address format](#server-address-format)
-  section for more information on the format of the string.
+  section for more information on the format of the string. This field is
+  deprecated in favor of [server_join](#server_join).
+
+  Note that `retry_join` can be defined for only servers as a command-line
+  flag (clients are only able to define via the client configuration).
+
+   ```sh
+   $ nomad agent -retry-join "127.0.0.1:4648"
+   ```
 
 - `retry_interval` `(string: "30s")` - Specifies the time to wait between retry
-  join attempts.
+  join attempts. This field is  deprecated in favor of
+  [server_join](#server_join).
 
 - `retry_max` `(int: 0)` - Specifies the maximum number of join attempts to be
   made before exiting with a return code of 1. By default, this is set to 0
-  which is interpreted as infinite retries.
+  which is interpreted as infinite retries. This field is  deprecated in favor
+  of [server_join](#server_join).
 
 - `start_join` `(array<string>: [])` - Specifies a list of server addresses to
   join on startup. If Nomad is unable to join with any of the specified
   addresses, agent startup will fail. See the
   [server address format](#server-address-format) section for more information
-  on the format of the string.
+  on the format of the string. This field is  deprecated in favor of
```

Comment:
I wonder if it is worth adding a header 3 right above `Server Address Format` called `Deprecated Parameters` and move these old fields there.

Reply:
I think it is a good idea to show more clearly which fields are deprecated, will do this.